aws-for-fluent-bit.yaml

```diff
@@ -0,0 +1,86 @@
+package:
+  name: aws-for-fluent-bit
+  version: 2.31.12.20230629
+  epoch: 0
+  description: AWS provides a Fluent Bit image with plugins for both CloudWatch Logs and Kinesis Data Firehose.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - fluent-bit
+      - go
+      - aws-flb-kinesis-compat
+      - aws-flb-firehose-compat
+      - aws-flb-cloudwatch-compat
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+
+vars:
+  FLB_DOCKER_BRANCH: "1.8"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aws/aws-for-fluent-bit
+      expected-commit: dfb7a7f63eadbe46a846cfbff05e57948154d39c
+      tag: v${{package.version}}
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/etc/ecs
+      mkdir -p "${{targets.destdir}}"/etc/fluent-bit/etc
+      mkdir -p "${{targets.destdir}}"/etc/fluent-bit/parsers
+      cp -r ./ecs/* ${{targets.destdir}}/etc/ecs
+      cp fluent-bit.conf "${{targets.destdir}}"/etc/fluent-bit/etc/
+      cp ./entrypoint.sh "${{targets.destdir}}"/etc/fluent-bit/entrypoint.sh
+      cp AWS_FOR_FLUENT_BIT_VERSION "${{targets.destdir}}"/etc/fluent-bit/AWS_FOR_FLUENT_BIT_VERSION
+      chmod +x "${{targets.destdir}}"/etc/fluent-bit/entrypoint.sh
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/etc/fluent-bit/configs
+      mv configs/parse-json.conf \
+        configs/minimize-log-loss.conf \
+        configs/output-metrics-healthcheck.conf \
+        configs/plugin-metrics-to-cloudwatch.conf \
+        configs/plugin-and-storage-metrics-to-cloudwatch.conf \
+        configs/plugin-metrics-parser.conf \
+        "${{targets.destdir}}"/etc/fluent-bit/configs/
+
+  - working-directory: fluent-bit-docker-image
+    pipeline:
+      - uses: git-checkout
+        with:
+          repository: https://github.com/fluent/fluent-bit-docker-image
+          branch: ${{vars.FLB_DOCKER_BRANCH}}
+          expected-commit: 98ce3316ea93751ddd33bc319a8f9d177493155b
+      - runs: |
+          cp conf/parsers*.conf "${{targets.destdir}}"/etc/fluent-bit/etc
```

Comment:
Are we sure this is right?  We are copying the same files to two different paths?

Reply:
Yep, that's weird. But this is what they did: https://github.com/aws/aws-for-fluent-bit/blob/69366b130a28e14c3c9548eabd69b499bb128c7b/scripts/dockerfiles/Dockerfile.build#L61-L64

Reply:
Ok, just wanted to be sure.